entrypoint.sh

```diff
@@ -153,7 +153,7 @@ echo -n "" >/etc/nginx/nginx.manifest.caching.config.conf
     # First tier caching of manifests; configure via MANIFEST_CACHE_PRIMARY_REGEX and MANIFEST_CACHE_PRIMARY_TIME
     location ~ ^/v2/(.*)/manifests/${MANIFEST_CACHE_PRIMARY_REGEX} {
         set \$docker_proxy_request_type "manifest-primary";
-        set \$cache_key \$uri;
+        set \$request_method \$cache_key \$uri;
```

Comment:
why are you writing to the $request_method variable? you should only be reading it.